src/pages/lessons/[slug].tsx

```diff
@@ -80,10 +80,12 @@ export const getServerSideProps: GetServerSideProps = async function ({
       params && (await loadBasicLesson(params.slug as string))
 
     if (initialLesson && initialLesson?.slug !== params?.slug) {
-      res.setHeader('Location', initialLesson.path)
-      res.statusCode = 302
-      res.end()
-      return {props: {}}
+      return {
+        redirect: {
+          destination: initialLesson.path,
+          permanent: false,
```

Comment:
this should be true!

Reply:
I'll switch it to true.

I think it originally should have been 301 instead of 302 -- I did know that distinction off the top of my head, so looked it up https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302